website/client/src/components/group-plans/createGroupModalPages.vue

```diff
@@ -1,3 +1,4 @@
+ <!-- THIS IS A VERY OLD FILE DO NOT USE -->
```

Comment:
If this file shouldn't be used, can we delete it?

Reply:
Yeah, I was gonna delete it when everything was done